tests/test_renderer.py

```diff
@@ -1,11 +1,14 @@
+import os
 import pytest  # type: ignore
+from xml.dom import minidom  # type: ignore
 
 import sphinx.addnodes
 import sphinx.environment
 from breathe.parser.compound import (
     compounddefTypeSub, linkedTextTypeSub, memberdefTypeSub, paramTypeSub,
     refTypeSub, MixedContainer
 )
+from breathe.parser.compoundsuper import sectiondefType
```

Comment:
Any reason for not using `sectiondefTypeSub`?

Reply:
It was more complicated for this test.